source/Core/ModuleInstaller.php

```diff
@@ -672,5 +672,7 @@ protected function resetCache()
         if ($this->getModuleCache()) {
             $this->getModuleCache()->resetCache();
         }
+        //clear utilsObject internal cache
+        UtilsObject::resetModuleVariables();        
```

Comment:
The method UtilsObject::resetModuleVariables(); is deprecated. Please use ModuleVariablesLocator::resetModuleVars instead.  See [UtilsObject.php#L382](https://github.com/OXID-eSales/oxideshop_ce/blob/64b64a29e670acd92fc2e55323df9b37ef07b64d/source/Core/UtilsObject.php#L382)